packages/datasources/trino/package.json

```diff
@@ -10,7 +10,7 @@
 	},
 	"dependencies": {
 		"@evidence-dev/db-commons": "workspace:*",
-		"presto-client": "^0.13.0"
+		"presto-client": "^1.1.0"
```

Comment:
This is the changelog since
```
Versions
1.1.0:
add automatic retries for server errors
follow redirects if servers simply redirect client's request
fix bug of HTTP/HTTPS procotol handling
fix bug about cancelling queries
1.0.0:
add test cases and CI setting, new options and others, thanks to the many contributions from Matthew Peveler (@MasterOdin)
add "timeout" option to retry requests for server errors
change "error" callback (or "callback" as fallback path) to be specified when "success" is used
```

I was hitting the http/https bug.